executioncontext/workspacecontext/src/main/java/org/jetbrains/bsp/bazel/workspacecontext/BuildManualTargetsSpec.kt

```diff
@@ -0,0 +1,25 @@
+package org.jetbrains.bsp.bazel.workspacecontext
+
+import io.vavr.control.Try
+import org.jetbrains.bsp.bazel.executioncontext.api.ExecutionContextSingletonEntity
+import org.jetbrains.bsp.bazel.executioncontext.api.ProjectViewToExecutionContextEntityMapper
+import org.jetbrains.bsp.bazel.projectview.model.ProjectView
+import org.jetbrains.bsp.bazel.projectview.model.sections.ProjectViewBuildManualTargetsSection
+
+data class BuildManualTargetsSpec(override val value: Boolean) : ExecutionContextSingletonEntity<Boolean>()
+
+private val defaultBuildManualTargetsSpec = BuildManualTargetsSpec(
+    value = false
+)
+
+internal object BuildManualTargetsSpecMapper : ProjectViewToExecutionContextEntityMapper<BuildManualTargetsSpec> {
+
+    override fun map(projectView: ProjectView): Try<BuildManualTargetsSpec> =
+        if (projectView.buildManualTargets == null) default()
+        else Try.success(map(projectView.buildManualTargets!!))
+
+    override fun default(): Try<BuildManualTargetsSpec> = Try.success(defaultBuildManualTargetsSpec)
+
+    private fun map(buildManualTargetsSection: ProjectViewBuildManualTargetsSection): BuildManualTargetsSpec =
```

Comment:
this method should be under `override fun map(projectView: ProjectView): Try<BuildManualTargetsSpec> =`